packages/cli/src/utils/foundry.ts

```diff
@@ -0,0 +1,24 @@
+import chalk from "chalk";
+import { execa } from "execa";
+
+async function safeExec(command: string, args: string[]): Promise<string> {
```

Comment:
this seems like a pretty generic wrapper, maybe it should be in its own file, like a replacement for execLog?

Reply:
"safe" is also a misleading term here (since executing arbitrary commands is ~rarely safe)

It looks like `execa` exports `execaSync` which seems to do what you're doing here, but maybe more robust:
https://github.com/sindresorhus/execa#execasyncfile-arguments-options

Reply:
Yeah not sure why this is called `safeExec`, should definitely be something like `execLog`, and agree with moving it to a general location.

Re `execaSync` - the one here is more of a helper to pipe the `stdout` and throw `stderr` in the main process (plus some default logging), while `execaSync` is just the non-async version of `execa`, so we'd still need the wrapper for setting the `stdio` options I think. Also being async is intended here, because it allows us to execute multiple commands without waiting for the result of the previous one.